bindings/nodejs/package.json

```diff
@@ -64,7 +64,7 @@
     "build:debug": "napi build --platform --target \"${NAPI_TARGET:-}\" --js generated.js && node ./scripts/header.js",
     "docs": "typedoc",
     "format": "prettier --write .",
-    "test": "cucumber-js",
+    "test": "SCHEME=memory vitest",
```

Comment:
You're still hard-coding `memory`. 
we'll have to get this externally. 
If you don't provide a scheme, skip the test.